### PR TITLE
Changes related to display title along with the table name while rend…

### PR DESCRIPTION
**Pull request by SharanyaMohan-30**
…ering

### CODE REVIEW

File: dataedit/templates/dataedit/dataview.html [hunk 1]
```diff
@@ -5,7 +5,8 @@
 
 
 {% block site-header %}
-  <h2 class="site-header"><a href="/dataedit/view/{{ schema }}">{{ schema }}</a>.{{ table }}</h2>
+  <h2 class="site-header">{{ title }}</h2>
```
Comment (MGlauer) [inline dataedit/templates/dataedit/dataview.html:8]: There should be a fallback, if there is no title set in the metadata string. I would recommend the following structure

{% if title %}
  <h3 class="site-header"><a href="/dataedit/view/{{ schema }}">{{ schema }}</a>: {{ title }}</h3>
{% else %}
  <h3 class="site-header"><a href="/dataedit/view/{{ schema }}">{{ schema }}</a>.{{ table }}</h3>
{% endif %}

File: dataedit/views.py [hunk 2]
```diff
@@ -917,6 +917,7 @@ def get(self, request, schema, table):
                 current_view = default
 
         table_views = list(chain((default,), table_views))
+        title_name = metadata["title"]
```
Comment (MGlauer) [inline dataedit/views.py:920]:
```suggestion
        title_name = metadata.get("title")
```
This does not throw an exception if the "title" field is missing

**Review by MGlauer — CHANGES_REQUESTED** [review]

**Comment by MGlauer** [comment]
Please use commit messages that conform to our [style guides](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md#branch-name-convention) and follow our [contribution guidelines](https://github.com/OpenEnergyPlatform/oeplatform/blob/develop/CONTRIBUTING.md#workflow) regarding the changelog etc.

File: dataedit/templates/dataedit/dataview.html (hunk 1 above)
Comment (SharanyaMohan-30) [reply dataedit/templates/dataedit/dataview.html:8]: Review Comment addressed.

File: dataedit/views.py (hunk 2 above)
Comment (SharanyaMohan-30) [reply dataedit/views.py:920]: Review Comment addressed.

**Comment by han-f** [comment]
Is there any indication when this will be completed? It would be great to have this issue https://github.com/OpenEnergyPlatform/oeplatform/issues/714 resolved for FNE 2021 next week, as we will guide though the platform in one workshop on thursday.

**Comment by christian-rli** [comment]
Is this still in progress? I checked into this branch and had a look at a table to which I added a name and a title. It is not shown on the page, or am I missing it? Also, this branch introduced a lot of warnings in django. Maybe this is related? 


api.UploadedImages: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the ApiConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
axes.AccessAttempt: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
axes.AccessLog: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
captcha.CaptchaStore: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
dataedit.Filter.value: (fields.W904) django.contrib.postgres.fields.JSONField is deprecated. Support for it (except in historical migrations) will be removed in Django 4.0.
	HINT: Use django.db.models.JSONField instead.
dataedit.Filter: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the DataeditConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
dataedit.Schema: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the DataeditConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
dataedit.Table: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the DataeditConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
dataedit.TableRevision: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the DataeditConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
dataedit.Tag: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the DataeditConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
dataedit.View.options: (fields.W904) django.contrib.postgres.fields.JSONField is deprecated. Support for it (except in historical migrations) will be removed in Django 4.0.
	HINT: Use django.db.models.JSONField instead.
dataedit.View: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the DataeditConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
login.ActivationToken: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
login.GroupMembership: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
login.GroupPermission: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
login.UserPermission: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
login.myuser: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the AppConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
modelview.BasicFactsheet: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the ModelviewConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
modelview.Energyscenario: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the ModelviewConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
modelview.Energystudy: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the ModelviewConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.
tutorials.Tutorial: (models.W042) Auto-created primary key used when not defining a primary key type, by default 'django.db.models.AutoField'.
	HINT: Configure the DEFAULT_AUTO_FIELD setting or the TutorialsConfig.default_auto_field attribute to point to a subclass of AutoField, e.g. 'django.db.models.BigAutoField'.

**Comment by christian-rli** [comment]
Bildschirmfoto zu 2021-08-28 12-24-11

Screenshot of view that seemed unchanged to me.

**Comment by SharanyaMohan-30** [comment]
Hello @christian-rli thank you for the response .Title will only be displayed if there are any existing title provided for the table which can be verified if we click on Edit option(which is disabled in the above screenshot) under the Meta information . In this case I believe title is empty ,we have handled the scenario such that if title has not been provided we only display schema.tablename else schema : title.

**Pull request closed without merging**